sql/mob_pools.sql

```diff
@@ -6140,6 +6140,8 @@ INSERT INTO `mob_pools` VALUES (6096,'Frost_Flambeau','Frost_Flambeau',232,0x000
 INSERT INTO `mob_pools` VALUES (6097,'Ophiotaurus','Ophiotaurus',459,0x00006A0B00000000000000000000000000000000,1,1,7,200,100,0,1,1,1,0,0,0,1,129,4,0,0,1,0,1009);
 INSERT INTO `mob_pools` VALUES (6098,'Siren','Siren',44,0x0000320A00000000000000000000000000000000,1,1,7,200,100,0,1,1,1,0,0,32,131,129,0,0,0,0,0,1010);
 INSERT INTO `mob_pools` VALUES (6099,'Nekros_Hound','Nekros_Hound',143,0x00006C0100000000000000000000000000000000,1,1,3,240,100,0,1,0,0,0,0,0,396,133,0,0,0,1,0,143);
+INSERT INTO `mob_pools` VALUES (6100,'Eald_narche_3','Eald_narche_3',150,0x00000B0300000000000000000000000000000000,4,1,9,240,125,0,1,1,0,16,0,32,0,0,0,0,308,1,0,1012);
+INSERT INTO `mob_pools` VALUES (6101,'Kam_lanaut_2','Kam_lanaut_2',351,0x00001D0500000000000000000000000000000000,5,5,3,240,100,0,1,1,1,18,17,32,0,0,0,0,309,1,0,1011);
```

Comment:
Now that the skill and spell list IDs have been adjusted in `mob_skill_lists` and `mob_spell_list`, they'll need to be updated in the original `mob_pools` too~!